src/renderer/components/+workloads-overview/overview-statuses.tsx

```diff
@@ -5,72 +5,54 @@ import { observer } from "mobx-react";
 import { Trans } from "@lingui/macro";
 import { OverviewWorkloadStatus } from "./overview-workload-status";
 import { Link } from "react-router-dom";
-import { cronJobsURL, daemonSetsURL, deploymentsURL, jobsURL, podsURL, statefulSetsURL } from "../+workloads";
-import { podsStore } from "../+workloads-pods/pods.store";
-import { deploymentStore } from "../+workloads-deployments/deployments.store";
-import { daemonSetStore } from "../+workloads-daemonsets/daemonsets.store";
-import { statefulSetStore } from "../+workloads-statefulsets/statefulset.store";
-import { jobStore } from "../+workloads-jobs/job.store";
-import { cronJobStore } from "../+workloads-cronjobs/cronjob.store";
+import { resourceURL, resourceStores } from "../+workloads";
 import { namespaceStore } from "../+namespaces/namespace.store";
 import { PageFiltersList } from "../item-object-list/page-filters-list";
 import { NamespaceSelectFilter } from "../+namespaces/namespace-select";
-import { isAllowedResource } from "../../../common/rbac";
+import { isAllowedResource, KubeResource, ResourceNames } from "../../../common/rbac";
+import { autobind } from "../../utils";
+import { _i18n } from "../../i18n";
+
+const resources: KubeResource[] = [
+  "pods",
+  "deployments",
+  "statefulsets",
+  "daemonsets",
+  "jobs",
+  "cronjobs",
+]
 
 @observer
 export class OverviewStatuses extends React.Component {
+  @autobind()
+  renderWorkload(resource: KubeResource): React.ReactElement | null {
+    const store = resourceStores[resource];
+    const items = store.getAllByNs([...namespaceStore.contextNs])
+    const name = _i18n._(ResourceNames[resource])
```

Comment:
This wont work. 
You should always provide texts directly to `_i18n._` or `<Trans>` calls (wrap it in `ResourceNames`)

Reply:
Also when you introduce new localization strings to code base don't forget to run `yarn i18n:extract` before commit.